$MirOS: ports/textproc/p5-XML-SAX/patches/patch-Makefile_PL,v 1.2 2009/01/01 17:34:57 bsiegert Exp $
--- Makefile.PL.orig	Mon Aug  4 23:06:58 2008
+++ Makefile.PL	Wed May  6 20:02:49 2009
@@ -16,38 +16,6 @@ sub MY::install {
     package MY;
     my $script = shift->SUPER::install(@_);
 
-    # Only modify existing ParserDetails.ini if user agrees
-
-    my $write_ini_ok = 0;
-
-    eval { require XML::SAX };
-    if ($@) {
-        $write_ini_ok = 1;
-    }
-    else {
-        my $dir = File::Basename::dirname($INC{'XML/SAX.pm'});
-        if (-e File::Spec->catfile($dir, 'SAX', 'ParserDetails.ini')) {
-            $write_ini_ok =
-                ExtUtils::MakeMaker::prompt(
-                    "Do you want XML::SAX to alter ParserDetails.ini?", "Y"
-                ) =~ /^y/i;
-        }
-        else {
-            $write_ini_ok = 1;
-        }
-    }
-    
-    if ($write_ini_ok) {
-        $script =~ s/install :: (.*)$/install :: $1 install_sax_pureperl/m;
-        $script .= <<"INSTALL";
-
-install_sax_pureperl :
-\t\@\$(PERL) -MXML::SAX -e "XML::SAX->add_parser(q(XML::SAX::PurePerl))->save_parsers()"
-
-INSTALL
-
-    }
-
     return $script;
 }
 
